client/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "wall",
+  "version": "1.0.0",
+  "description": "a very simple app that allows user to write things on a wall.",
+  "main": "index.js",
+  "scripts": {
+    "watch": "webpack --watch",
+    "start": "webpack-dev-server --open --config webpack.dev.js",
```

Comment:
@EvansHG There were no documentation on how to set-up the client module. I did the usual `npm i` and then `npm start`:

![image](https://user-images.githubusercontent.com/3919995/61643314-92293700-ac70-11e9-991a-3adb9fd67c28.png)